src/about_alan.md

```diff
@@ -1,20 +1,19 @@
+To accomplish this, `alan` makes one single, significant trade-off versus other programming languages: Functions in `alan` are all acyclic directed graphs. No arbitrary loops or recursion are allowed.
 
-[^1]: Technically an escape hatch through the event loop has been left open for very awkward recursive calls. This does not impact the advantages of the runtime for the language, however, and can be useful if absolutely necessary, but please reach out if you feel the need to use it, as that means either you're doing something wrong, or there's something missing to the language that should be added.
-[^2]: http://www.brendangregg.com/ebpf.html
-[^3]: Again, there is an intentional escape hatch, but it should be considered an emergency-only usage.
-[^4]: Assuming no bugs in the runtime and no mistakes in our reasoning. This hasn't been formally verified, so I may be wrong, but it doesn't appear that way after a lot of ad-hoc analysis.
-
-What you get in return for accepting these hopefully minor restrictions is a language that can automatically scale to your workload without rewriting, ever. Since all code is guaranteed[^5] to halt, it is possible to predict its runtime, and therefore to decide which thread(s) on which machine(s)[^6] should execute any particular part of it.
+This does not mean that you can't loop over data or write recursive algorithms, just that they are provided through controlled built-in functions that the compiler and runtime can reason about to provide automatic parallelization when possible, or to force handling a recursion error instead of crashing on a stack overflow.
 
-[^5]: Guarantees not guaranteed! ;)
-[^6]: This will only be available from the proprietary SaaS service at first, and eventually a proprietary enterprise compiler will be available to license once less-predictable datacenters are handled by the runtime. The open source runtime will only decide which threads to use for any given task, making it more like Go, but with less insulting assumptions that developers are idiots.
+This means that the code that you write in `alan` is not *quite* Turing-complete[^1]. But we believe that we have cut "close enough" to Turing-completeness and provided enough controlled mechanisms to fill the gaps that the advantages in having predictable functions which allows the runtime to be able to judge when parallelization makes sense based on the data to be processed and the complexity of the code to be executed outweighs the few places where `alan`'s syntax is slightly more awkward than its peers'.
```

Comment:
nit: can we break the 2nd sentence into two?

Reply:
Sure.